### ISSUE TITLE
Error while compiling the example

### ISSUE THREAD

**Issue by cloutiy** [nim-lang/ui#5]
I've built libui and installed ui with nimble install ui.

But when trying to compile the example I get the following error:

```
Hint: used config file '/etc/nim.cfg' [Conf]
Hint: system [Processing]
Hint: test [Processing]
Hint: ui [Processing]
Hint: rawui [Processing]
Hint: math [Processing]
/home/drifter/.nimble/pkgs/ui-0.9/ui/rawui.nim(24, 12) Error: string literal expected
```

**Comment by Araq** [comment]
You need nim devel (v0.16.1) for this to work.

**Comment by EffeErre** [comment]
So maybe `ui.nimble` should be corrected as 0.9 version doesn't work with nim-0.16.0...

> # Dependencies
> 
> requires "nim >= 0.14.0"

**Comment by Araq** [comment]
I updated the dependencies to reflect reality.
